### PR TITLE
session-16 arrays

### PR DESCRIPTION
**Pull request by Fatmasiam**
- adding day schedule in` readme.md` file
- adding learning-outcomes
- explain the arrays, indexed, loops by @Fatmasiam 
- explain the array-methods and adding the homework by @alaa-yasin

### CODE REVIEW

**Comment by MohammedYehia** [comment]
@Fatmasiam is this ready for review?

**Comment by Fatmasiam** [comment]
@MohammedYehia Yes

File: coursebook/session-16/arrays.md [hunk 1]
```diff
@@ -101,3 +102,37 @@ arr[3](); // hello
...
+console.log( activities[1] ); //  ['Eating  ', 2]
+console.log( activities[3] ); //  ['Sleeping', 7]
+
+console.log( activities[1][1] ); // 2
+console.log( activities[0][0] ); // Working
+console.log( activities[3][0] ); // Sleeping
+
+activities[4] = ['Study   ', 10];
+ 
+now
```
Comment (MohammedYehia) [inline coursebook/session-16/arrays.md]: can you please convert this to a comment
```suggestion
// now 
```

File: coursebook/session-16/loops.md [hunk 2]
```diff
@@ -2,190 +2,106 @@
...
 do {
-  alert( i );
+  console.log(alpha[i]); // print the alpha[0], alpha[1]
   i++;
-} while (i < 3);
+} while (i < alpha.length);
 
 ```
-This form of syntax should only be used when you want the body of the loop to execute at least once regardless of the condition being truthy. Usually, the other form is preferred: while(…) {…}.
+**Refreshing Question** :keycap_star:  ==> separate[explain] the work for the previous example if the condition will  `i <= alpha.length`
```
Comment (MohammedYehia) [inline coursebook/session-16/loops.md]: could this be something like this
```suggestion
**Refreshing Question** :keycap_star:  what will happen if the condition was  `i <= alpha.length`
```

**Pull request merged**